features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -66,10 +66,12 @@
   "authentication.authenticator.basic.name": "BasicAuthenticator",
   "authentication.authenticator.basic.enable": true,
   "authentication.authenticator.basic.parameters.AuthMechanism": "basic",
+  "authentication.authenticator.basic.parameters.Tags": "Username-Password",
 
   "authentication.authenticator.basic_request_path.name": "BasicAuthRequestPathAuthenticator",
   "authentication.authenticator.basic_request_path.enable": true,
   "authentication.authenticator.basic_request_path.parameters.AuthMechanism": "basic",
+  "authentication.authenticator.basic_request_path.parameters.Tags": "Request path",
```

Comment:
better to use a unified format for tags. if we follow the pattern in "Username-Password" to use '-' as delimiter and start each word with a capital letter, then better to use that pattern in all the places

Reply:
Addressed with commit ad110cdff4ffcfe5766b31f1a09bebaf633730e6